docs/source/addons/theme.md

```diff
@@ -0,0 +1,220 @@
+---
+myst:
+  html_meta:
+    "description": "Create a theme add-on"
+    "property=og:description": "Create a theme add-on"
+    "property=og:title": "Create a theme add-on"
+    "keywords": "Volto, Plone, Semantic UI, CSS, Volto theme"
+---
+
+# Create a theme add-on
+
+We can create a Volto Add-on that acts as a theme Add-on, so we can detach it from the project.
+The advantage is that you can deploy the same theme in different projects, or have themes depending on conditions that you could inject on build time.
+For convenience, it can also be set via a `THEME` environment variable.
+
+1. Add a `theme` key in your `volto.config.js` file in the root of your project:
```

Comment:
When do I need a`volto.config.js`? Maybe not to explain here in theme.md, but in /addons/index.md.

Reply:
It's explained here: https://github.com/plone/volto/blob/master/docs/source/configuration/volto-config-js.md 
Maybe I should point people there to know what's about it?

Reply:
Ah, haven't seen. Well explained!